src/fragmentarium/ui/display/Transliteration.tsx

```diff
@@ -201,26 +201,37 @@ const lineComponents: ReadonlyMap<
   ['CompositeAtLine', DisplayDollarAndAtLine]
 ])
 
+const rulingsToNumber: ReadonlyMap<string, number> = new Map([
+  ['SINGLE', 1],
+  ['DOUBLE', 2],
+  ['TRIPLE', 3]
+])
+
 function DisplayRulingDollarLine({
   line,
   container = 'div'
 }: {
   line: Line
   container?: string
 }): JSX.Element {
-  const rulingDollarLine = line as RulingDollarLine
-  const element = 'Transliteration__ruling'
+  const rulingLine = line as RulingDollarLine
+  const rulingType = `Transliteration__ruling--${rulingLine.number.toLowerCase()}`
+  const rulingsNumber = rulingsToNumber.get(rulingLine.number) as number
   return React.createElement(
     container,
-    { className: `Transliteration__${rulingDollarLine.type}` },
-    <hr
-      className={classNames([
-        element,
-        `${element}--${rulingDollarLine.number.toLowerCase()}`
-      ])}
-    />
+    { className: 'Transliteration__RulingDollarLine' },
+    [...Array(rulingsNumber).keys()].map((value: number) => {
```

Comment:
What's this? It is hard to see what it is supposed to do.

Reply:
@yCobanoglu please, do not resolve conversations without addressing the issue in some way

The line is very cryptic. You should think about ways to make it easier to understand.

Reply:
It seems that you are creating a range of numbers. Extracting a method with a descriptive name would make that more obvious, or maybe [someone already implemented such a method](https://lodash.com/docs/4.17.15#range).